deploy/kiali/kiali_cr.yaml

```diff
@@ -716,6 +716,10 @@ spec:
 #  ---
 #  kiali_feature_flags:
 #
+# Flag to indicate Kiali to enable/disable clustering and federation related features.
+#    ---
+#    clustering: true
```

Comment:
I have two comments here.

1. Is this a good name for this? The name "clustering" seems generic. It may be the proper name for this feature, but the word "clustering" doesn't bring to my mind a specific feature here. In our [Features](https://kiali.io/documentation/latest/features/#_advanced_mesh_deployment_and_multi_cluster_support) doc page I see words like "multi-cluster" and "multi-network" and "mesh". Should this be called "multi_clustering"? I just want to make sure we think about this name before committing to support it - if "clustering" is the correct name for this feature, leave it.

2. This multi-cluster support ("clustering") is becoming a major feature in Kiali. I think there are going to be many more pieces of this large feature set that we may want to be configurable. Let's prepare for this future by having this "clustering" be a parent to a potential group of settings (in the same way we have ui_defaults here):
```
kiali_feature_flags:
  clustering:
    enabled: true
```

This way in the future we can support additional settings that go under the clustering heading.

Otherwise, I can easily see this section get out of hand with a bunch of independent and separate clustering settings - we do not want to have something like this happen:
```
kiali_feature_flags:
  clustering: true
  clustering_cache_size: 5
  clustering_secrets_edit: foo
  clustering_allow_rbac: true
```

It would be better for everything to go under the clustering heading:
```
kiali_feature_flags:
  clustering:
    enabled: true
    cache_size: 5
    secrets_edit: foo
    allow_rbac: true
```

Reply:
Well, "clustering" is the term that has been used in some calls.

TBH, I don't know why that term is being used. "Clustering" is usually used in machine learning. By googling, I see this definition: "Clustering is the task of dividing the population or data points into a number of groups such that data points in the same groups are more similar to other data points in the same group than those in other groups." Although the definition may give a close idea to multi-cluster, I'm not sure if it's really OK to use that term.

About (2), I'm working on it.